### PR TITLE
Completed all missing translations to french

### PR DESCRIPTION
**Pull request by Hexabitt**
I don't know to which mailing list i should send the info, so i hope someone will see this

### CODE REVIEW

**Comment by mattock** [comment]
Do we have anyone who could review this?

File: res/openvpn-gui-res-fr.rc [hunk 1]
```diff
@@ -143,50 +143,50 @@ CAPTION "Général"
...
-    AUTORADIOBUTTON "On connect", ID_RB_BALLOON1, 28, 155, 50, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "On connect/reconnect", ID_RB_BALLOON2, 86, 155, 90, 10
-    AUTORADIOBUTTON "Never", ID_RB_BALLOON0, 181, 155, 40, 10
+    GROUPBOX "Préférences", 202, 6, 82, 235, 90
+    AUTOCHECKBOX "Ajouter au fichier log", ID_CHK_LOG_APPEND, 17, 95, 60, 10
+    AUTOCHECKBOX "Afficher la fenêtre des scripts", ID_CHK_SHOW_SCRIPT_WIN, 17, 110, 200, 10
+    AUTOCHECKBOX "Connexion silencieuse", ID_CHK_SILENT, 17, 125, 200, 10
+    LTEXT "Afficher une bulle d'information", ID_TXT_BALLOON, 17, 140, 100, 10
+    AUTORADIOBUTTON "A la connexion", ID_RB_BALLOON1, 28, 155, 50, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "A la connexion/déconnexion", ID_RB_BALLOON2, 86, 155, 90, 10
```
Comment (mattock) [inline res/openvpn-gui-res-fr.rc]: Should this be "A la connexion/reconnexion"?

DISCLAIMER: I'm doing this review semi-blindfolded based on English and Italian :smile:

**Comment by mattock** [comment]
@Hexabitt I forgot to mention that you sent your modification to the exact right place :smile: .

I added one line note. I also asked if someone on openvpn-devel mailing list could do a proper language review. If nobody does, we can still merge this using a lazy-ACK ("Nobody cared enough, so in it goes...").

File: res/openvpn-gui-res-fr.rc (hunk 1 above)
Comment (20ton) [inline res/openvpn-gui-res-fr.rc]: Yes, reconnect should be "reconnexion"

File: res/openvpn-gui-res-fr.rc [hunk 2]
```diff
@@ -143,50 +143,50 @@ CAPTION "Général"
...
 ID_DLG_ADVANCED DIALOGEX 6, 18, 252, 218
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
-CAPTION "Advanced"
+CAPTION "Avancé"
 FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_FRENCH, SUBLANG_DEFAULT
 BEGIN
-    GROUPBOX "Configuration Files", 201, 6, 12, 235, 45
-    LTEXT "Folder:", ID_TXT_FOLDER, 17, 25, 32, 10
+    GROUPBOX "Fichier de configuration", 201, 6, 12, 235, 45
```
Comment (20ton) [inline res/openvpn-gui-res-fr.rc]: "Fichiers de configuration"

**Comment by mattock** [comment]
Mickael Chazaux on openvpn-user list noted a missing accent:

> Hi,
> 
> Reconnect would be "reconnexion", and the A should have an accent :
> À la connexion/reconnexion
> À la connexion
> 
> Cheers,
> 
> Mickael

File: res/openvpn-gui-res-fr.rc [hunk 3]
```diff
@@ -143,50 +143,50 @@ CAPTION "Général"
...
+    GROUPBOX "Fichiers log", 202, 6, 62, 235, 30
+    LTEXT "Dossier:", ID_TXT_FOLDER, 17, 74, 32, 10
     EDITTEXT ID_EDT_LOG_DIR, 53, 72, 150, 12, ES_AUTOHSCROLL
     PUSHBUTTON "…", ID_BTN_LOG_DIR, 208, 72, 25, 12
 
-    GROUPBOX "Script Timeout", 201, 6, 97, 235, 60
-    LTEXT "Preconnect script timeout:", ID_TXT_PRECONNECT_TIMEOUT, 17, 110, 100, 10
-    LTEXT "Connect script timeout:", ID_TXT_CONNECT_TIMEOUT, 17, 125, 90, 10
-    LTEXT "Disconnect script timeout:", ID_TXT_DISCONNECT_TIMEOUT, 17, 140, 90, 10
+    GROUPBOX "Timeout des scripts", 201, 6, 97, 235, 60
```
Comment (mandree) [inline res/openvpn-gui-res-fr.rc]: Is there a good French translation for "Timeout"?

Comment (20ton) [inline res/openvpn-gui-res-fr.rc]: There is no exact translation. 
"Délai d'attente" or "Dépassement de délai" is good for a socket, but here I would translate the group title to "Temps d’exécution maximum des scripts" and drop the redundant "timeout" in each entry (better) or use a simple "délai"
But someone who knows what a script is will also know "timeout", so I think it's fine and that's why I didn't comment on this (I don't like over-translation)

**Comment by Hexabitt** [comment]
Thank you for the review.
Corrected the little mistakes and followed the recommendations of @20ton.
Committed the changes to my existing branch.

**Comment by mattock** [comment]
Ready to merge?

**Pull request merged**